packages/types/src/signUp.ts

```diff
@@ -59,6 +59,7 @@ export interface SignUpResource extends ClerkResource {
   createdSessionId: string | null;
   createdUserId: string | null;
   abandonAt: number | null;
+  legalAcceptedAt: number | null;
```

Comment:
how come this is not a Date ? i thought that was the pattern

Reply:
on User.ts you are converting this to a Date

Reply:
Yeah it's a timestamp, the type is a number as we don't parse it like we do on `User` resource